### DISCUSSION TITLE
skilljack essence and cost

### DISCUSSION THREAD

**Discussion by ernstmann** [chummer5a/chummer5a#4706] [General]
Hi guys, 
1st up, you are doing a great job! Love the chummer generators.
Just noticed the the cost for the skilljack in Headware are set to 10.000 but actually are 1000/ rating, additionally the essence should be stable with 0,1 but is now going up with the ratings.

cheers
ernstmann

**Comment by chummer5a** [comment]
That's only true in the German rules; Chummer's data uses English RAW. You can enable the German Data Changes to configure this behaviour on your own.
